vernac/vernacentries.ml

```diff
@@ -935,8 +935,6 @@ let polymorphic_cumulative ~is_defclass =
     (bool_attribute ~name:"polymorphic"
      ++ bool_attribute ~name:"cumulative")
   >>= fun (poly,cumul) ->
-  if is_defclass && Option.has_some cumul
-  then user_err Pp.(str "Definitional classes do not support the inductive cumulativity attribute.");
```

Comment:
Where has this check gone? Was it dead code?

Reply:
we now only parse the attributes we actually need